components/docs-chef-io/content/automate/ha_cert_deployment.md

```diff
@@ -17,158 +15,190 @@ gh_repo = "automate"
 {{% automate/ha-warn %}}
 {{< /warning >}}
 
-Custom certificates can be added during deployment for different services.
+Custom certificates can be added during deployment for different services. Follow the steps below to provide custom certificates using the `config gen` command.
 
-## Setting common certificates for all the nodes of a service
+## Setting Standard Certificates for all the Nodes of a Service
 
-The following services support custom certificates:
+During the config generation, the following prompts will ask for custom certificates to provide relevant certificate paths for the services.
 
-### Common custom certificate for Chef Automate
+### Common Custom Certificate for Chef Automate
 
-To add a common custom certificate for all the nodes of the Chef Automate service, add the configurations given below in the `config.toml` file before running the `chef-automate deploy` command:
+To add a standard custom certificate for all the nodes of Chef Automate services, select:
 
-```toml
-[automate.config]
-enable_custom_certs = true
+```bash
+  Will you use custom certs for any service like Automate, Chef Infra Server, PostgreSQL, or OpenSearch:
+    no
+  > yes
+```
```

Comment:
Do we need to show this ?
Can we just explain technical terms asked in the CLI, rather than the flow explained.